File: Modul 5/Bahasa C/PRAK501-2310817120001-DinaIzzatiElfadheya.c
#include <stdio.h> 

int MaxBilangan(int bil1, int bil2, int bil3, int bil4) {
    int max = bil1;
    if (bil2 > max) {
        max = bil2;
    }
    if (bil3 > max) {
        max = bil3;
    }
    if (bil4 > max) {
        max = bil4;
    }
    return max;
}
int main() {
    int a, b, c, d;
    printf("Input\n");
    scanf("%d %d %d %d", &a, &b, &c, &d);
    int hasil = MaxBilangan(a, b, c, d); 
    printf("\nOutput\n");
    printf("%d", hasil);
    return 0;
}

File: Modul 5/Bahasa Python/PRAK501-2310817120001-DinaIzzatiElfadheya.py
def MaxBilangan(bil1, bil2, bil3, bil4):
    max_bilangan = bil1
    if bil2 > max_bilangan:
        max_bilangan = bil2
    if bil3 > max_bilangan:
        max_bilangan = bil3
    if bil4 > max_bilangan:
        max_bilangan = bil4
    return max_bilangan

a, b, c, d = map(int, input("Input\n").split())
print("\nOutput")
hasil = MaxBilangan(a, b, c, d)
print(hasil)
